package.json

```
@@ -2,16 +2,35 @@
   "name": "root",
   "private": true,
   "license": "Apache-2.0",
+  "type": "module",
   "devDependencies": {
+    "@eslint/js": "^9.23.0",
     "concurrently": "9.1.0",
     "husky": "^9.1.6",
     "typescript": "^5.7.3",
     "vitest": "^3.0.7",
     "@vitest/ui": "^3.0.7",
-    "lint-staged": "^15.4.3"
+    "lint-staged": "^15.4.3",
+    "eslint": "^9.23.0",
+    "eslint-config-prettier": "^10.1.1",
+    "eslint-plugin-cypress": "^4.2.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
+    "eslint-plugin-perfectionist": "^4.10.1",
+    "eslint-plugin-prettier": "~5.2.6",
+    "eslint-plugin-react": "^7.37.4",
+    "eslint-plugin-react-hooks": "^5.2.0",
+    "eslint-plugin-sonarjs": "^3.0.2",
+    "eslint-plugin-testing-library": "^7.1.1",
+    "eslint-plugin-xss": "^0.1.12",
+    "prettier": "~3.5.3",
+    "typescript-eslint": "^8.29.0",
+    "@typescript-eslint/eslint-plugin": "^8.29.0",
+    "@typescript-eslint/parser": "^8.29.0",
+    "@linode/eslint-plugin-cloud-manager": "^0.0.10",
+    "jiti": "^2.4.2"
   },
   "scripts": {
-    "lint": "eslint . --quiet --ext .js,.ts,.tsx",
+    "lint:all": "concurrently -n manager,api-v4,validation,ui,utilities,queries,shared -c green,blue,yellow,magenta,cyan,gray,blue \"pnpm run --filter linode-manager lint\" \"pnpm run --filter @linode/api-v4 lint\" \"pnpm run --filter @linode/validation lint\" \"pnpm run --filter @linode/ui lint\" \"pnpm run --filter @linode/utilities lint\" \"pnpm run --filter @linode/queries lint\" \"pnpm run --filter @linode/shared lint\"",
```

We may be able to use a cleaner pnpm commend for this:

```suggestion
    "lint:all": "pnpm run --filter '*' --parallel lint",
```

I would also be okay with keeping the command name `lint`, but I don't have a strong opinion

---

see this [thread](https://github.com/linode/manager/pull/11983#discussion_r2031794570) - need to use the recursive flag here too, will update

`lint:all` was deliberate because every because can `lint` individually so i feel like it makes the intent clearer?